src/ontology/foodon-edit.ofn

```diff
@@ -20657,6 +20660,32 @@ AnnotationAssertion(rdfs:label obo:FOODON_00004502 "fish (raw)"@en)
 EquivalentClasses(obo:FOODON_00004502 ObjectIntersectionOf(obo:FOODON_03411222 ObjectSomeValuesFrom(obo:RO_0000053 obo:FOODON_00004348)))
 SubClassOf(obo:FOODON_00004502 obo:FOODON_03411222)
 
+# Class: obo:FOODON_00004503 (wild animal)
+
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_00004503 "An undomesticated animal specie."@en)
+AnnotationAssertion(obo:IAO_0000119 obo:FOODON_00004503 "https://en.wikipedia.org/wiki/Wildlife"^^xsd:anyURI)
+AnnotationAssertion(dcterms:contributor obo:FOODON_00004503 <https://orcid.org/0000-0001-5275-8866>)
+AnnotationAssertion(dcterms:date obo:FOODON_00004503 "2023-10-06T13:00:00Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:FOODON_00004503 "wild animal")
+SubClassOf(obo:FOODON_00004503 obo:FOODON_03411297)
+
+# Class: obo:FOODON_00004504 (seabird)
+
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_00004504 "A bird that is adapted to live within the marine environment."@en)
+AnnotationAssertion(obo:IAO_0000119 obo:FOODON_00004504 "https://en.wikipedia.org/wiki/Seabird"^^xsd:anyURI)
+AnnotationAssertion(dcterms:contributor obo:FOODON_00004504 <https://orcid.org/0000-0001-5275-8866>)
+AnnotationAssertion(dcterms:date obo:FOODON_00004504 "2023-10-06T13:11:05Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:FOODON_00004504 "seabird")
+SubClassOf(obo:FOODON_00004504 obo:FOODON_00002616)
+
+# Class: obo:FOODON_00004505 (wild bird)
+
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_00004505 "A bird other than domestic poultry and caged bird."@en)
```

Comment:
Perhaps just define similar to "wild animal" since its a subclass? Aristotelean "A wild animal which is a bird".  Or perhaps "An undomesticated bird".  This allows for instance level determination, e.g. wild pigeon vs. domestic pigeon.  Even though USDA categorizes pigeons as poultry, they can be wild too and this becomes important to state in biosamples.

Reply:
> Perhaps just define similar to "wild animal" since its a subclass? Aristotelean "A wild animal which is a bird". Or perhaps "An undomesticated bird". This allows for instance level determination, e.g. wild pigeon vs. domestic pigeon. Even though USDA categorizes pigeons as poultry, they can be wild too and this becomes important to state in biosamples.

'An undomesticated bird' makes more sense then. I haven't added that as a subclass of 'wild animal', I think  I should do that as well. Thanks